source/features/pr-jump-to-first-unviewed-file.tsx

```diff
@@ -0,0 +1,34 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function jumpToFirstUnviewed(): void {
+	const firstNotView = select('.js-reviewed-checkbox:not([checked])')!;
+	if (!firstNotView) {
+		// Scroll to page end
+		window.scrollTo(0, document.body.scrollHeight);
```

Comment:
Let's use the current scroll position instead of `0`. Does this work in both browsers?
```suggestion
		window.scrollTo(window.scrollX, document.body.scrollHeight);
```

Reply:
Checking